gateway/gateway-runtime/python-executor/executor/server.py

```
@@ -0,0 +1,425 @@
+    ) -> proto.DestroyPolicyResponse:
+        """Synchronous destroy — runs in thread pool."""
+        instance = self._store.remove(request.instance_id)
+        if instance is None:
+            logger.warning(
+                f"DestroyPolicy: instance_id={request.instance_id} not found (already destroyed?)"
+            )
+            return proto.DestroyPolicyResponse(success=True)
+
+        try:
+            instance.close()
+        except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)